indra/explanation/pathfinding/util.py

```diff
@@ -76,45 +83,59 @@ def signed_nodes_to_signed_edge(source, target):
         return None, None, None
 
 
-def get_sorted_neighbors(G, node, reverse, force_edges=None):
-    """Sort the returned neighbors in descending order by belief
+def get_sorted_neighbors(
+        G: nx.DiGraph,
+        node: Node,
+        reverse: bool,
+        force_edges: Optional[List[Edge]] = None,
+        edge_filter: Optional[Callable[[nx.DiGraph, Node, Node], bool]] = None
+) -> List[Node]:
+    """Filter and sort neighbors of a node in descending order by belief
 
     Parameters
     ----------
     G : nx.DiGraph
         A networkx DiGraph
-    node : str|int
-        A valid networkx node name
+    node : Node
```

Comment:
You can just skip all of these type annotations in text if they're done as type annotations in the definition (everything should work as expected except for the Returns, that needs to be duplicated for napoleon style docs)

Reply:
That is one of the things I tagged you for, I wasn't sure exactly how it was working :) Will fix